samples/winapi_drivelist/main.c

```diff
@@ -56,11 +65,30 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
 
-    while(1) {
+    goto unmount_c;
+    goto unmount_e;
```

Comment:
This should be removed, you're bypassing the `Sleep`-loop here and the second goto is dead code.